### ISSUE TITLE
Output formats for check blocks

### ISSUE THREAD

**Issue by jpolitz** [brownplt/pyret-lang#51]
We should have machine-readable output formats for test results, e.g. --check-json.

**Comment by blerner** [comment]
Is this still as urgent, given the new load-lib lets us just `getModuleResultChecks` as a value?

**Comment by Emmay** [comment]
Is this still an issue, as per Ben's comment?

**Comment by blerner** [comment]
Yes.  @jswrenn is implementing something like this, separate from Pyret itself, and we ought to incorporate some of that work back into Pyret.

**Comment by schanzer** [comment]
@blerner @jswrenn do we still want to keep this open?

**Comment by schanzer** [comment]
@jswrenn I can only assume that examplar was the intended recipient. :)

Has any of this been incorporated back into Pyret?

**Comment by jswrenn** [comment]
> @jswrenn I can only assume that examplar was the intended recipient. :)

It was actually for [*Who Tests the Testers*](https://cs.brown.edu/~sk/Publications/Papers/Published/wkf-who-tests-testers/paper.pdf) and some pyret autograding that used the same infrastructure! That work involved writing a [custom `runner.js`](https://github.com/brownplt/powder-monkey/blob/master/evaluate/runner.js) to produce JSON test output. Unfortunately, this work never made it back into pyret-lang.
